addons/binding/org.openhab.binding.seneye/README.md

```diff
@@ -17,7 +17,7 @@ You can have multiple seneye devices in your home, just make sure that your aqua
 
 ## Discovery
 
-Discovery is not supported, the seneye monitor must be configured manually
+Discovery is not supported, the seneye monitor must be configured manually; either through Paper or .items
```

Comment:
I'm not sure you should add this. It's also possible with other ui's, or name it correctly `PaperUI`.

Reply:
Leave a comment

> I have added Signed-off-by but it is still it still appears to be failing that test.
> Can you let me know how to fix this?
> 
> Maybe your commit is actually committed with a different user email, but if you can be reached on the address in your sign off its okay.

I have ensured I am using the same email address to login, as I am signing but this is still failing.
However it is a valid email address, so this shouldn't be a problem.